### PR TITLE
Added Tracing MS

### PR DESCRIPTION
**Pull request by ayancovich**

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -102,7 +102,15 @@ services:
...
      ports:
-        - 5986:5986       
-         
+        - 5986:5986   
+        
+  pad-b-tracing: 
+     build: ../tracing/
+     depends_on:
+       - pad-b-registry
+     ports:
```
Comment (nestorreloaded) [inline docker-compose.yml:112]: We are using the 8000 porte range, I think

**Review by nestorreloaded — CHANGES_REQUESTED** [review]

File: docker-compose.yml (hunk 1 above)
Comment (exe135) [reply docker-compose.yml:112]: Yep, we are using this range on the others, from 8080 increasing 1 in each service.

**Review by exe135 — APPROVED** [review]

File: docker-compose.yml (hunk 1 above)
Comment (nestorreloaded) [reply docker-compose.yml:112]: Maybe we can make tracing run on 8090?

Comment (ayancovich) [reply docker-compose.yml:112]: ok

**Review by nestorreloaded — APPROVED** [review]

**Pull request merged**